src/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "GOV.UK Toolkit",
   "description": "Switch between GOV.UK environments and content",
   "homepage_url": "https://github.com/alphagov/govuk-toolkit-chrome",
-  "version": "1.8.0",
+  "version": "1.9.0",
```

Comment:
@tijmenb I've removed the commit where I upgraded the extension. I will do that in a separate PR.